app/assets/stylesheets/core/_type.scss

```diff
@@ -1,8 +1,8 @@
-html, body, h1, h2, h3, h4, h5,
+html, body, h1, h2, h3, h4, h5, h6
 ul, ol, li, blockquote, label, form,
 input, span, a {
   font-family: 'Roboto', Helvetica, Arial, sans-serif;
-  color: $black;
+  color: $shaleDark;
```

Comment:
I generally prefer not to use straight black for significant amounts of body text.

Reply:
Did you mean, you prefer to use straight black?

Reply:
I prefer to not use black for body text. IMO it's a little bit easier on the eyes for reading long chunks of body text if its a little bit lighter (e.g. https://designforhackers.com/blog/impressionist-color-theory/)

Reply:
Nevermind I am confused.